### PR TITLE
Features/movies list

### PR DESCRIPTION
**Pull request by krzysztof-gronski**
popular movies list default page

### CODE REVIEW

File: src/common/Tile/index.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+
+export const Tile = () => {
+  return (
```
Comment (ursmal1948) [inline src/common/Tile/index.js:4]: Maybe rename to a movieTile? I think we will also have a seperate personTile so we can differentiate it

**Review by ursmal1948 — APPROVED** [review]

File: src/common/Tile/index.js (hunk 1 above)
Comment (krzysztof-gronski) [reply src/common/Tile/index.js:4]: i am going to create a common component with separate options

**Comment by ulyanalakevich** [comment]
Good job! Don't you think it's better to make a separate file App.js for App.css , and the index.js file will intended for global things like Provider and Global Style?🤔

**Comment by krzysztof-gronski** [comment]
Actually App.css is removed in the newest version, any styles will be put in styled.js file intended for this component ;)

File: src/common/MainContainer/index.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
+import { Container, TilesContainer, Header } from "./styled";
+import { Tile } from "../Tile";
+import { sampleTiles } from "./sampleTiles";
+
+export const MainContainer = () => {
+  return (
+    <Container>
+      <Header>Popular movies</Header>
+      <TilesContainer>
+        {sampleTiles.map((tile) => (
```
Comment (ursmal1948) [inline src/common/MainContainer/index.js:10]: Maybe we should also add a unique key for each rendered item? key=tile.id

Comment (krzysztof-gronski) [reply src/common/MainContainer/index.js:10]: sure, I'm surprised it didn't cause any error

File: src/common/MainContainer/index.js [hunk 3]
```diff
@@ -15,6 +15,7 @@ export const MainContainer = () => {
             genres={tile.genres}
             rate={tile.rate}
             votesNr={tile.votesNr}
+            key={tile.id}
```
Comment (ursmal1948) [inline src/common/MainContainer/index.js:18]: now looks good!

**Review by ursmal1948 — APPROVED** [review]

**Comment by krzysztof-gronski** [comment]
First ever merge in history confirmed!

**Pull request merged**